### PR TITLE
chore: aether@1.7.2 + gather@3.4.2

### PR DESCRIPTION
**Pull request by obdulia-losantos**
- Docker containers:
  - Aether 1.7.2
  - Gather 3.4.2
  - Keycloak 9.0.3
- Producer accesses kernel via RETFul API.
- No service uses the `postgres` user to access the database.
- Integration tests check producer kernel access via RESTFul API and via Database.
- Remove kernel dependency while checking containers readiness.

### CODE REVIEW

File: aether/init.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
```
Comment (lordmallam) [inline aether/init.sh:3]: 2020? 😄

File: connect/init.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
```
Comment (lordmallam) [inline connect/init.sh:3]: same

File: aether/init.sh (hunk 1 above)
Comment (obdulia-losantos) [reply aether/init.sh:3]: I will change all the copyrights in another PR :grimacing:

File: connect/init.sh (hunk 2 above)
Comment (obdulia-losantos) [reply connect/init.sh:3]: same comment :upside_down_face:

**Pull request merged**